firewood/src/merkle.rs

```diff
@@ -1726,6 +1801,21 @@ mod tests {
         assert_eq!(encoded, new_node_encoded);
     }
 
+    #[test_case(leaf(Vec::new(), Vec::new()) ; "empty leaf encoding")]
+    #[test_case(leaf(vec![1, 2, 3], vec![4, 5]) ; "leaf encoding")]
+    #[test_case(branch(b"value".to_vec(), vec![1, 2, 3].into()) ; "branch with chd")]
+    #[test_case(branch(b"value".to_vec(), None); "branch without chd")]
+    #[test_case(branch(Vec::new(), None); "branch without value and chd")]
+    fn node_encode_decode_(node: Node) {
```

Comment:
q: what does the trailing _ mean here?

Reply:
Was following the test name of the other suite. Maybe @richardpringle can jump in here? :)